pkg/storage/snapshot/restore.go

```diff
@@ -276,6 +278,10 @@ func (ctrl *VMRestoreController) updateVMRestore(vmRestoreIn *snapshotv1.Virtual
 		return 0, ctrl.doUpdateError(vmRestoreIn, err)
 	}
 	if updated {
+		if err := ctrl.updateRestorePVCOwnership(vmRestoreOut, target); err != nil {
```

Comment:
Any reason why shouldn't do this in the restore target Reconcile()?

Reply:
No, its better there.